Formula/hq.rb

```diff
@@ -18,7 +23,9 @@ class Hq < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", *std_cargo_args
+    cd "html-query" do
+      system "cargo", "install", *std_cargo_args
+    end
```

Comment:
this would be better
```suggestion
    system "cargo", "install", *std_cargo_args(path: "html-query")
```